content/en/events/banners/ambient-release.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Ambient Alpha Release
+period_start: latest_release
+period_duration: 30
+max_impressions: 4
+link: https://istio.io/latest/news/releases/1.18.x/announcing-1.18/#ambient-mesh
+---
+
+Ambient is part of Istio {{< istio_version >}} as alpha status, try it out!
```

Comment:
```suggestion
{{< gloss "ambient" >}}Ambient{{< /gloss >}} is part of Istio {{< istio_version >}} as alpha status, [try it out!](docs/ops/ambient/getting-started/)
```

Reply:
The entire banner is a link.

Reply:
Ah!

Reply:
Was thinking if the whole banner should link to the getting started page, or the alpha announcement.